packages/components/tests/integration/components/hds/form/field/index-test.js

```diff
@@ -59,6 +61,7 @@ module('Integration | Component | hds/form/field/index', function (hooks) {
           <F.Label>This is the label</F.Label>
           <F.HelperText>This is the helper text</F.HelperText>
           <F.Control><pre id={{F.id}} aria-describedby={{F.ariaDescribedBy}}>This is a mock control</pre></F.Control>
+          <F.CharacterCount>20/40</F.CharacterCount>
```

Comment:
should we include a test for the `dynamic` version? essentially, testing that these two lines do what we expect them to do: https://github.com/hashicorp/design-system/blob/6d1df7431e26bbc58d42e1473678cd19c5f17d76/packages/components/addon/components/hds/form/field/index.hbs#L34-L35

Reply:
for the `Form::Field` all tests are using a mock-control (not a real `<input>`) so we can't have a dynamic version there. the two properties highlighted above (the `controlId` and appending descriptors) are tested in https://github.com/hashicorp/design-system/pull/1802/files#diff-7293ac7c93f148221709265bfb8098aefc04d3e239ba7f8c8fd88d6a54561490R81-R85